src/shared/lib/analytics/server.ts

```diff
@@ -35,11 +35,6 @@ export const setupAnalytics = async (options?: Props) => {
 
   return {
     track: (options: { event: string } & PostEventPayload["properties"]) => {
-      if (process.env.NODE_ENV !== "production") {
-        console.log("Track", options);
-        return;
-      }
```

Comment:
was there a specific reason for that change?
I'd actually prefer to keep it active locally it helps test events during development. Let me know if there's a context I missed

Reply:
Yes, but this code checks the environment, not whether analytics are actually enabled

Analytics could be enabled locally during development (which is what I personnaly do to test events). The `NODE_ENV` check it ensures we don't accidentally send real events from dev to prod dashboards, but still logs the event payloads for local debugging.

unless we're explicitly filtering dev environments upstream or mocking the analytics entirely, I think this early return is useful to keep let me know what you think